storage/range.go

```diff
@@ -686,8 +687,8 @@ func (r *Range) addWriteCmd(args proto.Request, reply proto.Response, wait bool)
 		// If the original client didn't wait (e.g. resolve write intent),
 		// log execution errors so they're surfaced somewhere.
 		if !wait && err != nil {
-			log.Warningf("non-synchronous execution of %s with %+v failed: %s",
-				args.Method(), args, err)
+			log.CtxWarningf(ctx.Add(logfield.Error, err),
```

Comment:
The `net/context` docs suggest giving each key a different type which is somewhat more efficient for `interface{}` comparison which compares types first. Doubt it matters in practice.